modules/nlu/src/backend/application/scoped/bot.ts

```diff
@@ -67,10 +67,6 @@ export class Bot implements Trainer, Predictor {
       const stringId = this._modelIdService.toString(modelId)
       throw new Error(`Model ${stringId} not found on file system.`)
     }
-    return this._load(model)
```

Comment:
yeah... I think there used to be 2 users of this private method and now there's only one and I forgot to get rid of it